### PR TITLE
Albert Zhang assignment 2

### PR DESCRIPTION
**Pull request by AlbeartoZ-hang**

### CODE REVIEW

File: Assignment2.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Albert Zhang
+1.
+1) x <= 4 || (x != 9  && x > 10)
+5 <= 4 || (5 != 9  && 5 > 10)
+5 <= 4 || (true && false)
+false || false
+false
```
Comment (oldclesleycode) [inline Assignment2.txt:7]: Right! 💯

File: Assignment2.txt [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+2) x > 0 && x < 10 && x != 6
+5 > 0 && 5 < 10 && 5 != 6
+true && true && true
+true
```
Comment (oldclesleycode) [inline Assignment2.txt:12]: Perfect! 💯

File: Assignment2.txt [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+3) x == 1 || x > 0
+5 == 1 || 5 > 0
+false || true
+true
```
Comment (oldclesleycode) [inline Assignment2.txt:17]: Perfect! 💯

File: Assignment2.txt [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+2. You can use neither as a switch test condition because the switch test condition can only be a character or an integer
+
+3.There is an error because there is no & before the userIn in line 2. As written, this will cause a compile error.
```
Comment (oldclesleycode) [inline Assignment2.txt:21]: Perfect! 💯

File: Assignment2.txt [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+5 > 0 && 5 < 10 && 5 != 6
+true && true && true
+true
+
+3) x == 1 || x > 0
+5 == 1 || 5 > 0
+false || true
+true
+
+2. You can use neither as a switch test condition because the switch test condition can only be a character or an integer
```
Comment (oldclesleycode) [inline Assignment2.txt:19]: Right! 👍

File: evenodd.c [hunk 6]
```diff
@@ -0,0 +1,18 @@
+/*Checks if user input is odd or even */
+#include <stdio.h>
+int main () {
+	int input;
+	printf("Enter a number:");
+	scanf("%d", &input);
+	if (input%2 == 0) {
+		printf("The number is even \n");
+	}
+	if (input%2 != 0) {
```
Comment (oldclesleycode) [inline evenodd.c:10]: Since this isn't an else if statement, "that's not even a number" is printed whenever a number is even. Nice try at trying to handle all input though!

File: threenums.c [hunk 7]
```diff
@@ -0,0 +1,29 @@
+#include <stdio.h>
+int main ()  {
```
Comment (oldclesleycode) [inline threenums.c:2]: No comments and you didn't properly handle the case where two numbers are equal and also the greatest number (so, say, 9 9 1). To avoid mishandling edge cases, test your code thoroughly!
